src/translations/es.json

```diff
@@ -23,5 +23,12 @@
     "content2": "Sé el primero en alcanzar 21 puntos en esta explosiva batalla de ingenio. Monta una ofensiva valiente mientras interrumpes a tu oponente con trucos malvados. ¿Tienes lo necesario para convertirte en el Señor de las Profundidades?",
     "readRules": "Leer las reglas",
     "joinDiscord": "Únete a nuestra comunidad en Discord"
+  },
+  "lobby":{
+    "lobbyFor": "Lobby para",
+    "ready": "Preparado",
```

Comment:
Issue: Value for 'ready' is all caps in `en.json` Keep these casing consistent between locales.